edweiss-app/app/(app)/courses/[id]/materials/[materialId]/index.tsx

```diff
@@ -261,7 +263,7 @@ const DocumentScreen: ApplicationRoute = () => {
     // ------------------------------------------------------------
     const displayPageNumber = () => (
         <TView testID={testIDs.pageNumView} backgroundColor='transparent' radius={5} px={10} py={5} style={{ position: 'absolute', top: 10, left: 10, backgroundColor: 'rgba(128, 128, 128, 0.2)' }}>
-            <TText testID={testIDs.pageNumText} bold color='text'>
+            <TText testID={testIDs.pageNumText} bold color='constantBlack'>
```

Comment:
Yes actually it looks good because the display number in placed on the PDF. The pages of PDFs are white so even in dark mode, we have to display in black. Do you see what I mean ?